src/josepy/json_util.py

```diff
@@ -157,6 +158,8 @@ class Foo(object):
 
     """
 
+    _fields = {}  # type: Dict[str, Any]
```

Comment:
Since josepy requires Python 3.6+, I think we can replace this and all other instances of mypy `# type:` comments with PEP484 type hints, as you mentioned:

```suggestion
    _fields: Dict[str, Any] = {}
```

There's not that many in the project, maybe we can do it now?

Reply:
Yes. I'll run https://github.com/ilevkivskyi/com2ann on the project :)